demo/index.ts

```diff
@@ -41,8 +41,7 @@ const config: IGenerateSwaggerConfig = {
 			authMiddlewareName: 'authenticate'
 		}
 	},
-	tags: {},
-	filter: '.*users.*'
+	tags: {}
```

Comment:
@kot-1999 why did you delete filter from demo ?

Reply:
Because of only "user" endpoints were included to the result API doc.

Reply:
@kot-1999 it is a demo. It was demonstration of filter feature 🥲